ingest-folder-opex-creator/src/test/scala/uk/gov/nationalarchives/ingestfolderopexcreator/LambdaTest.scala

```diff
@@ -69,6 +69,20 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
         .willReturn(ok().withBody(queryResponse))
     )
 
+  private def runAndGetBody(skipIngest: Boolean): String = {
+    stubBatchGetRequest(dynamoGetResponse())
```

Comment:
Maybe the method could be called something like `getFolderOpexFromS3Request`?

Reply:
That is a lot more descriptive :smile: I've changed it.